### DISCUSSION TITLE
See outside of the page border when zooming.

### DISCUSSION THREAD

**Discussion by Desdeldo** [xournalpp/xournalpp#3482] [Q&A] [answered]
Is it possible to move the page more to the left, overcoming the page limit border? Or the page view is always limited to the page border limit when zooming?

**Comment by LittleHuba** [comment, marked as answer]
No that is currently not possible. This will be part of the redesign of our rendering engine.

**Reply by Desdeldo** [reply]
Ohh, I see... I will be waiting then, this is the best app that I found for what I need, I liked it a lot. Thanks!
